.gitattributes

```diff
@@ -3,6 +3,10 @@
 ###############################################################################
 * text=auto encoding=UTF-8
 
+# csc/vbc are shell scripts and should always have unix line endings
+src/Compilers/CSharp/CscCore/csc text eol=lf
+src/Compilers/VisualBasic/VbcCore/vbc text eol=lf
```

Comment:
Can you elaborate why this was necessary here? There are several other shell scripts in the repo that aren't listed here.  Do they suffer from whatever issue prompted this change?